treeherder/model/models.py

```diff
@@ -336,57 +335,6 @@ class Meta:
         db_table = 'failure_classification'
 
 
-# TODO: Remove these as final phase of Bug 1387640
-
-class JobExclusion(models.Model):
-
-    """
-    A filter represents a collection of properties
-    that you want to filter jobs on. These properties along with their values
-    are kept in the info field in json format
-    """
-    name = models.CharField(max_length=255, unique=True)
-    description = models.TextField(blank=True)
-    info = JSONField()
-    author = models.ForeignKey(User)
-
-    class Meta:
-        db_table = 'job_exclusion'
```

Comment:
The `GRANT` entries for these tables can also be removed from `docs/admin.rst`

Reply:
Dang!  I remembered seeing those in a couple of my searches, but somehow missed them.  Once again, nice catch...  :)